info.json

```diff
@@ -237,6 +249,18 @@
 				"link": "https://github.com/python-restx/flask-RESTX",
 				"info": ["Fork of Flask-RESTPlus: Fully featured framework for fast, easy and documented API development with Flask."],
 				"tags": ["swagger", "marshmallow", "api"]
+			},
+			{
+				"name": "Flask-Restful",
```

Comment:
Could you remove flask-restful? Flask-restx is a maintained, swagger-included version of it.